packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -31,9 +31,13 @@ describe("TableFactory unit tests", () => {
 		class Column extends TableSchema.createColumn(schemaFactory, ColumnProps) {}
 
 		class RowProps extends schemaFactory.object("table-row-props", {
-			label: schemaFactory.optional(schemaFactory.string),
+			selectable: schemaFactory.optional(schemaFactory.boolean),
```

Comment:
Small semantic update to the test case to make optional props make a bit more sense.

Reply:
optional selectability seems a bit odd. That leaves three cases (undefined, true, false). Maybe put a doc comment on this field (to show that works and produces useful intelisense, and also to make it clear what the 3 states mean?)

Reply:
Yeah, I debated how much we wanted to document these "simple" examples. But you're right: the semantics aren't clear. I'll add something brief.